code/game/objects/items/storage/book.dm

```diff
@@ -65,6 +65,22 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			dat += {"<tr><td><img src="[nicename]"></td><td><a href="?src=[REF(src)];seticon=[i]">[nicename]</a></td></tr>"}
 		dat += "</table></body></html>"
 		H << browse(dat, "window=editicon;can_close=0;can_minimize=0;size=250x650")
+	else if(H.mind.holy_role)
```

Comment:
This isn't chaplain, those are all people that belong in a sect

Reply:
2 is the main chaplain and 1 is the secondary chaplains (latejoins / ERT)